IntegrationTestDS3/IntegrationTestDS3Client.cs

```diff
@@ -178,6 +178,7 @@ public void GetSystemInfo()
         **/
 
         [Test]
+        [Ignore]
```

Comment:
Do we really want to ignore these?  It seems like they should would be better to skip them for 3.0 rather than ignore them outright.